### PR TITLE
[enhancement] 60 desk available

### LINKED ISSUE #60: API to check if desk is available for booking on a particular date
## NEW API:
/api/desk-service/availability
POST

requestBody:
{
"orgId": "string"
"floorId":"string"
"requestDate": "string" dd/MM/yyyy

} 

Action: api will call booking-service- get all the booking for particular data on orgId and floorId. API will return a desk which is not already booked in booking service.

### PR DESCRIPTION
**Pull request by devjuliet**
**API to check if desk is available for booking on a particular date**
- [x] Added endpoint POST /availability
- [x] call booking-service- get all the booking for particular date on orgId and floorId, and siteId. 
- [x] API will return a desk which is not already booked in booking service

Note: in the deployment deskId is not in Booking. But in local yes

Closes #60

### CODE REVIEW

**Comment by kanwalnainsingh** [comment]
@devjuliet build is failing can you check

**Comment by devjuliet** [comment]
Yes, I will check 😄

File: code_base/micro-services/desk-service/src/main/java/github/opendesk/deskservice/client/BookingClient.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+import java.util.List;
+
+@Service
+public class BookingClient {
+
+    @Autowired
+    RestTemplate restTemplate;
+
+    String bookingServiceUrl = "https://www.bookopendesk.com/api/booking-service";
```
Comment (kanwalnainsingh) [inline code_base/micro-services/desk-service/src/main/java/github/opendesk/deskservice/client/BookingClient.java:18]: @devjuliet can you move this from const to properties.

**Review by kanwalnainsingh — COMMENTED** [review]
added minor comment please fix rest all looks good.

File: code_base/micro-services/desk-service/src/main/java/github/opendesk/deskservice/client/BookingClient.java (hunk 1 above)
Comment (devjuliet) [reply code_base/micro-services/desk-service/src/main/java/github/opendesk/deskservice/client/BookingClient.java:18]: Yes, let me check

**Comment by devjuliet** [comment]
@kanwalnainsingh It is done 😺

**Pull request merged**